presto-docs/src/main/sphinx/connector/druid.rst

```diff
@@ -0,0 +1,26 @@
+===============
+Druid Connector
+===============
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+The Druid Connector allows querying `Apache Druid <https://druid.apache.org/>`_ data from Presto.
+
+Configuration
+-------------
+
+To configure the Druid connector, create a catalog properties file
+``etc/catalog/druid.properties`` with the following contents, replacing
```

Comment:
somehow need to express that this is just an example name ..

Reply:
Agreed, but this needs to be fixed across all connectors